test/Integration.hs

```diff
@@ -0,0 +1,39 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeApplications #-}
+
+-- | Swarm integration tests
+module Main where
+
+import Control.Monad
+import Data.Either
+import Data.Functor
+import Data.Text (Text)
+import System.Directory (doesFileExist, listDirectory)
+import System.FilePath.Posix
+import Test.Tasty
+import Test.Tasty.HUnit
+import Witch
+
+import Swarm.Language.Pipeline (processTerm)
+
+main :: IO ()
+main = do
+  paths <- acquire
+  defaultMain . exampleTests $ paths
+
+exampleTests :: [(FilePath, String)] -> TestTree
+exampleTests inputs = testGroup "Test example" (map exampleTest inputs)
+
+exampleTest :: (FilePath, String) -> TestTree
+exampleTest (path, fileContent) = do
+  testCase ("processTerm for contents of " ++ show path) $
+    assertBool "should return Right value" (isRight . processTerm $ into @Text fileContent)
```

Comment:
Could you please use `assertEqual` or something else that would show the parse failure?

I had to use `swarm format example/zigzag.sw` to see what was actually the issue.
IMHO "it should be `Right`" does not help anybody.